### PR TITLE
alphonse: round10 — lr=1.5e-4 + EMA=0.999 (push LR ceiling, single delta)

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

PR #115 (current SOTA, test 10.580) used lr=1e-4 — 2× the prior lr=5e-5. thorfinn explicitly noted in their results comment that:
- final gradient norms at ep9 were 0.5 with **0% clipping** (no binding)
- val curve was still **descending at ep9** (Δep8→ep9 = −0.25, not asymptoting)
- "Push lr further: 1.5e-4 or 2e-4" listed as #3 follow-up

Earlier PR #112 (alphonse, lr=1e-4 alone WITHOUT EMA=0.999) showed ep8-9 plateau overshoot — but PR #115's EMA=0.999 smooths that overshoot, possibly opening LR headroom up to 1.5e-4 or 2e-4.

**This run tests the LR ceiling.** If lr=1.5e-4 + EMA=0.999 holds (no overshoot), follow-up will be lr=2e-4. If it diverges or overshoots, we know lr=1e-4 is the ceiling for the 9-epoch budget.

Projected outcomes:
- **Best:** lr=1.5e-4 captures more loss landscape per step → ep9 val ~9.0-9.2 → test ~10.0-10.3 (−2 to −5% vs SOTA)
- **Likely:** marginal improvement from lr step → ep9 val ~9.3-9.4 → test ~10.4-10.5
- **Worst:** ep8-9 overshoot returns → val plateaus at ~9.6-9.8 → test ~10.7+ regression

## Instructions

Single delta from PR #115 SOTA — only `--lr` changes:

```bash
torchrun --standalone --nproc_per_node=8 train.py \
  --agent alphonse --optimizer lion --lr 1.5e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 8 --model-slices 128 \
  --ema-decay 0.999 \
  --wandb-group tay-round10-lr1p5e4
```

**Single delta vs PR #115 SOTA:** `--lr 1.5e-4` (was `1e-4`). All other flags identical to PR #115.

Hard timeout 270 min. Hard cap 9 epochs.

Post results: best val_primary/abupt_axis_mean_rel_l2_pct, final test_primary/abupt_axis_mean_rel_l2_pct, per-component test breakdown, W&B run URL, training time per epoch, gradient norm trajectory (especially watch for ep8-9 overshoot signal).

Key observations to report:
- Did training stay smooth (no overshoot at late epochs)?
- Was ep8→ep9 still descending, or plateauing?
- Final gradient norms (vs PR #115's 0.5 with 0% clipping)?

## Baseline (PR #115 thorfinn — current SOTA, test_abupt 10.580)

| Metric | PR #115 SOTA | AB-UPT |
|---|---:|---:|
| `test_primary/abupt_axis_mean_rel_l2_pct` | **10.580** | — |
| best val | 9.484 | — |
| ep9 trajectory: 53.75/24.15/16.51/13.47/11.83/10.88/10.16/9.73/**9.48** | | |

W&B run: `d03oghpp`. Reference — PR #112 alphonse lr=1e-4 alone (no EMA): test 11.324 (ep8-9 overshoot), val curve flat at ep9.

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #148 because ADVISOR: Closing this run early. After 4 epochs the lr=1.5e-4 trajectory is stably ~40% behind SOTA with no sign of compression:

| Epoch | alphonse #148 val | PR #115 SOTA val | Δ |
|---|---:|---:|---:|
| ep0 | 72.43 | 53.75 | +34.8% |
| ep1 | 34.24 | 24.15 | +41.8% |
| ep2 | 22.73 | 16.51 | +37.7% |
| ep3 | 18.96 | 13.47 | **+40.8%** |

The behind-pct is **stable, not compressing**, which means lr=1.5e-4 is not just slow to start — it's converging to a worse asymptote than lr=1e-4. Projected final val ~13-14 → test ~14.8 (significantly worse than SOTA 10.580).

**Why lr=1.5e-4 lost on tay's compound stack:**
- Lion update rule is `sign(momentum_blend)`, so effective step magnitude is `lr` directly (no √v scaling like AdamW). Doubling LR doubles the step, with no adaptive damping.
- T_max=50 cosine barely decays over 9 epochs (~92% of lr_max at ep9), so the noise from higher LR persists throughout training.
- EMA=0.999 has half-life ~10 epochs, which can smooth gradient noise but cannot reverse converging to a worse local minimum.
- Compound result PR #115 already crossed an inflection — lr=1e-4 was the sweet spot, and the gradient evidence (still descending at ep9) suggested room *up to* 1e-4 but not necessarily beyond.

**Closing-time signal for the project**: The "push LR ceiling" lever is closed at 1.5e-4. The ceiling for Lion+EMA=0.999 in this 9-epoch budget is between 1e-4 and 1.5e-4 — and the 5e-5 → 1e-4 step (PR #115's −5%) was probably the only available LR-ceiling gain.

**Freeing this GPU for fresh hypothesis.** New assignment incoming..
